source/documentation/API-reference.html.md.erb

```diff
@@ -46,6 +46,66 @@ This explains the steps shown in the above diagram on the access flow for Trader
 
 The Intermediary or agent does not receive the access token directly. The application uses it on their behalf.
 
+## Testing your Trader Goods Profile integration
+
+### Using the Trader Goods Profile sandbox environment
+
+The TGP sandbox provides a mix of dynamic and static behaviour, along with an Admin API. This combination of test endpoints allows you to test the creation and subsequent updates to goods records over the course of trader’s use of TGP.
+
+### Dynamic endpoints
+
+The TGP sandbox contains a test data store allowing you to create and manipulate data as part of the user journey. The following endpoints support dynamic behaviour:
+
+- Create a TGP goods record
+- Update a TGP goods record
+- Get TGP goods records
+- Get a single TGP goods record
+- Remove a TGP goods record
+- Maintain TGP profile
+
+Records created and updated within the data store will persist for one year.
+
+Schema validation is applied to the endpoints in the same way as the production endpoints, so attempting to create or update a record with invalid data will result in an error being returned.
+
+### Creating a new TGP for test users in the dynamic sandbox
+
+In the <u>sandbox environment only</u>, the Maintain TGP Profile endpoint can be used to create new TGPs for test users you’ve set up in the [Developer Hub][DEV_HUB].
+
+To create a new TGP, you’ll need to use the test user’s EORI number as the eori in the request as well as the actorId. You’ll also need to provide a number matching the required format for a UKIMS number to create the profile. Optionally you can also provide dummy data in the correct format for NIRMS or NIPHL numbers.
+
+Once created, you’ll be able to create and manipulate data for this test user’s EORI within the dynamic sandbox.
+
+*Note: the Maintain TGP Profile can not be used to create new profiles in the production environment.*
+
+### Static endpoints
+
+The following endpoints do not provide dynamic behaviour:
+
+- Request advice
+- Withdraw advice
```

Comment:
do we need to mention about Withdraw advice for now? if its not ready? @taylors-version ?

Reply:
good catch, nope, that should be removed otherwise it's confusing

Reply:
Removed withdraw advice